### PR TITLE
[web] Reduce code size impact of fallback font data

### PR DESCRIPTION
**Pull request by mdebbar**
Optimizes fallback font data introduced in https://github.com/flutter/engine/pull/35646.

### Counter app

Size of `main.dart.js`:
| | Before | After |
|-|-|-|
| Uncompressed | 1.65MB | 1.5MB |
| Gzip (-9) | 513KB | 483KB |
| Brotli (-9) | 419KB | 408KB |

Related to https://github.com/flutter/flutter/issues/113898

### CODE REVIEW

**Comment by mdebbar** [comment]
Do we want to move forward with this @hterkelsen @eyebrowsoffire ?

File: lib/web_ui/dev/generate_fallback_font_data.dart [hunk 1]
```diff
@@ -293,3 +301,23 @@ const List<String> fallbackFonts = <String>[
...
+String _packFontRanges(List<String> starts, List<String> ends) {
+  assert(starts.length == ends.length);
+
+  final StringBuffer sb = StringBuffer();
+
+  for (int i = 0; i < starts.length; i++) {
+    final int start = int.parse(starts[i], radix: 16);
+    final int end = int.parse(ends[i], radix: 16);
+
+    sb.write(start.toRadixString(36));
```
Comment (hterkelsen) [inline lib/web_ui/dev/generate_fallback_font_data.dart:314]: 36 just to get maximum "compression"?

**Review by hterkelsen — APPROVED** [review]
LGTM

File: lib/web_ui/dev/generate_fallback_font_data.dart (hunk 1 above)
Comment (mdebbar) [reply lib/web_ui/dev/generate_fallback_font_data.dart:314]: Yes. To encode the integer in as few characters as possible.

**Comment by eyebrowsoffire** [comment]
I tested this against the dart2wasm stuff and it doesn't cause anything to explode, so LGTM!

**Review by eyebrowsoffire — APPROVED** [review]
LGTM

**Comment by kevmoo** [comment]
@mdebbar – make non-draft and land?

**Pull request merged**